web/src/components/Features.js

```diff
@@ -79,7 +80,7 @@ const Features = () => {
           title="Open Source"
           url="https://github.com/wasp-lang/wasp"
           description={`
-            This is the way. Wasp is fully open-source and you're welcome to contribute!
+            Fully open-source and you’re welcome to contribute!
```

Comment:
Oh interesting, I don't even know how to type that character, the single quote thing that is not a single quote :D.